packages/grafana-ui/src/components/Logs/LogRowMessageDetectedFields.tsx

```diff
@@ -19,12 +19,11 @@ class UnThemedLogRowMessageDetectedFields extends PureComponent<Props> {
   render() {
     const { row, showDetectedFields, getFieldLinks, wrapLogMessage } = this.props;
     const fields = getAllFields(row, getFieldLinks);
-    const wrapClassName = cx(
-      wrapLogMessage &&
-        css`
-          white-space: pre-wrap;
-        `
-    );
+    const wrapClassName = wrapLogMessage
+      ? ''
+      : cx(css`
```

Comment:
I think we can remove `cx` from here, as we are not merging more classnames?

Reply:
Sure, definitely. Removed it 👍